src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -19,21 +20,31 @@ public static final class Drivetrain {
 	}
 	
 	public static final class Arm {
-		
+		// Array Indexes (Just to make things easier to read)
+        public static final int ARM = 0;
+		//Motor port
+		public static final int MOTOR_PORT1 = 7;
+		public final static int MOTOR_PORT2 = 8;
 		//all angles in rot here
 		public static final double intakeAngle = -.1;
 		public static final double ampAngle = .3;
 		public static final double speakerAngle = .4;
```

Comment:
don't make anyone waste an hour debugging because they thought it was in degrees instead of radians. Take a look at [WPILIB's new Java units](https://docs.wpilib.org/en/stable/docs/software/basic-programming/java-units.html)